AdminWebsite/AdminWebsite/Contracts/Enums/BookingStatus.cs

```diff
@@ -11,6 +11,6 @@ public enum BookingStatus
 
 public enum VideoSupplier
 {
-    Kinly = 1,
+    // Kinly = 1,
```

Comment:
are we intending to keep this commented for historical/documentation purposes? if so maybe put is as a summary docs on the enum else you'll get code smell alerts